src/Symfony/Bundle/TwigBundle/DependencyInjection/TwigExtension.php

```diff
@@ -151,4 +161,36 @@ public function getNamespace()
     {
         return 'http://symfony.com/schema/dic/twig';
     }
+
+    private function cleanUpDefinitionsMissingRequiredDependencies(ContainerBuilder $container)
+    {
+        if (!interface_exists('Symfony\Component\Templating\TemplateReferenceInterface')) {
+            $container->removeDefinition('twig.controller.exception');
+        }
```

Comment:
`findTemplate()` returns a `TemplateReferenceInterface` instance: https://github.com/symfony/symfony/blob/2.7/src/Symfony/Bundle/TwigBundle/Controller/ExceptionController.php#L99

Technically, this is not needed in 2.7 as the FrameworkBundle already requires the Templating component. But this check IMO is more correct as we do not rely on FrameworkBundle internals here.